src/main/webapp/WEB-INF/jsp/editDisplayPreferences.jsp

```diff
@@ -112,12 +112,12 @@
                 <div class="col-md-12">
                     <c:choose>
                         <c:when test="${not prefHideAbstract}">
-                            <input type="checkbox" value="true" name="hideAbstract"/>
-                            <label><spring:message code="edit.pref.hideabstract"/></label>
+                            <input role="checkbox" aria-labelledby="HideAbstract" type="checkbox" value="true" name="hideAbstract"/>
+                            <label id="HideAbstract"><spring:message code="edit.pref.hideabstract"/></label>
```

Comment:
I have found  a (possibly out-of-date) source about this matter, [here](https://www.w3.org/WAI/tutorials/forms/labels/), and basically, some assistive technology can be messed up by implicit labels, so, since it currently is functional, I think an implicit label should not be used.